channels/ht.m3u

```diff
@@ -1,7 +1,33 @@
+https://video1.getstreamhosting.com:1936/8055/8055/playlist.m3u8
+#EXTINF:-1 tvg-id="NETALKOLETV.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/OJbRkQ6.jpg" group-title="",NETALKOLE TV (180p)
+https://watch.haitilive.net/stream/netalkole/public/tv/index.m3u8
+#EXTINF:-1 tvg-id="RadioTélé4VEH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/1wuEyIF.jpg" group-title="Religious",Radio Télé 4VEH (720p)
+https://uni01rtmp.tulix.tv/4vehtv/4vehtv-firetv/playlist.m3u8
+#EXTINF:-1 tvg-id="RadioTélé4VEH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/1wuEyIF.jpg" group-title="Religious",Radio Télé 4VEH (720p)
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UClL_aynC_SESNLQEJHqO9MQ/live
+#EXTINF:-1 tvg-id="RadioTeleAmen.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/dJvJGDg.jpg" group-title="Religious",Radio Tele Amen FM (360p)
+http://184.173.179.163:1935/daniel/daniel/playlist.m3u8
+#EXTINF:-1 tvg-id="RadioTéléEclair.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/8ocCyo7.png" group-title="",Radio Télé Eclair (1080p)
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCOvGbDh5biuOqjx6G3CdrQg/live
+#EXTINF:-1 tvg-id="" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/QaWwNvo.jpg" group-title="Religious",Radio Télé Pitit Manman Mari (720p)
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCLeNHM8XDkZmd2rhV3ZG7Vg/live
+#EXTINF:-1 tvg-id="RTVC.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/Jp38Rzf.png" group-title="",Radio Télévision Caraïbes (RTVC) (720p)
+https://stream-02.nyc.dailymotion.com/sec(JtO5HO8PrOFTKKYNhfGSpkCAlv5YxyPWEpr2He2oJRA)/dm/3/x81lgjh/d/live-2.m3u8
+#EXTINF:-1 tvg-id="RadioTélévisionHirondelle.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/2gVnoSX.jpg" group-title="",Radio Télévision Hirondelle (480p)
+https://watch.haitilive.net/gethb/rtvh16/index.m3u8
+#EXTINF:-1 tvg-id="RadioTeleShalom.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/MSgSKym.png" group-title="",Radio Tele Shalom (720p) [Not 24/7]
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCBJ9zxns1hxblYZw4urBd_w/live
+#EXTINF:-1 tvg-id="RadioTelePlanetCompas.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/ddb4lsa.png" group-title="",Radio Tele Planet Compas (720p)
+https://5dcab9aed5331.streamlock.net/mrcompas1/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="TCH.ht" tvg-country="HT" tvg-language="French;Creole" tvg-logo="https://i.imgur.com/6LZVA4x.png" group-title="",Télé Conscience Haïtienne (TCH) (480p)
+https://tvlakay.haitilive.net/website/tchhaiti/player/mono.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/133883061-7be94c62-3a35-4024-b42f-1390c91ef921.png)

Reply:
The error will disappear if you change the protocol:

```suggestion
http://tvlakay.haitilive.net/website/tchhaiti/player/mono.m3u8
```